osfoffline/tasks/operations.py

```diff
@@ -132,6 +169,30 @@ def __init__(self, remote):
     @asyncio.coroutine
     def run(self):
         print("Delete Remote File: {}".format(self.remote))
+        db_parent = session.query(models.File).filter(models.File.osf_id == self.remote.parent.id).one()
+        path = os.path.join(db_parent.path, self.remote.name)
+
+        resp = yield from self.remote.request_session.request('DELETE', self.remote.id)
+        yield from resp.release()
+
+         # get item
+        try:
+            item = _get_item_by_path(path)
+        except ItemNotInDB:
```

Comment:
Let's try to avoid mixing error handling throughout the task. I'll put up my WIP PR in the moment, where I've added some error handling in a parent `run()` method that wraps the private implementation... basically, we want to be sure that our error handling provides a clean mechanism to send messages about problems to the UI.